### PR TITLE
www.tomshw.it

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/123443

<details><summary>Screenshot:</summary>

Снимок экрана 2022-07-07 в 13 13 28

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -138,6 +138,8 @@ voici.fr,the-sun.com,thesun.ie,capital.fr,ladbible.com,hitc.com,dualshockers.com
 !
 livoo.fr#$##tarteaucitronAlertBig { display: none!important; }
 livoo.fr#$#html { overflow: auto!important; }
+tomshw.it#$#html, body { overflow: auto !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:141]: 1. We do not add more than one selector to a single rule.
2. We already have the rules that change `overflow`.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -138,6 +138,8 @@ voici.fr,the-sun.com,thesun.ie,capital.fr,ladbible.com,hitc.com,dualshockers.com
...
+tomshw.it#$##iubenda-cs-banner { display: none !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:142]: Also add an exception for content blockers.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
Actually, it opens fine on my end. Maybe the current JS rule does not apply on Firefox? In that case, we should not fix anything, but we can give the rule to user.

**Comment by AdamWr** [comment]
Comments do not open if cookie notice is not accepted, like user described it here - https://github.com/AdguardTeam/AdguardFilters/issues/121221
I still can reproduce this issue.
With apps and extensions for Chrome/Edge this cookie is automatically closed, but extensions for Opera/Firefox cannot download JS rules, so cookie notice is not hidden.

**Comment by zloyden** [comment]
> extensions for Opera/Firefox cannot download JS rules

I've answered [here](https://github.com/AdguardTeam/AdguardFilters/issues/123443#issuecomment-1177377476).

**Pull request closed without merging**